proto/redpanda/api/dataplane/v1alpha1/secret.proto

```diff
@@ -98,7 +98,7 @@ message CreateConnectSecretRequest {
     }
   ];
 
-  // The Connect name.
+  // Name of secret.
```

Comment:
This is actually the Kafka Connect Connector name. https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/